setup.py

```diff
@@ -149,4 +149,5 @@ def get_long_description():
               },
           zip_safe=False,
           test_suite='selftests',
-          python_requires='>=2.6')
+          python_requires='>=2.6',
+          install_requires=['stevedore'])
```

Comment:
Actually, that was the intended scope of this first round.  We're working towards making remote, vm, docker and all of those optional plugins, with their own deps declared in their own `setup.py`.

And btw, thanks so much for testing it!